docs/src/content/docs/pt/index.mdx

```diff
@@ -13,7 +13,7 @@ hero:
       icon: right-arrow
       variant: primary
     - text: Ir para o desafio mais recente
-      link: /pt/challenges/signal/54-pipe-observable-to-signal/
+      link: /pt/challenges/signal/55-back-button-navigation/
```

Comment:
the path is not correct. you are inside angular folder and not signal. Need to be corrected in all files